package.json

```
@@ -1,5 +1,5 @@
 {
-  "version": "1.1.44",
+  "version": "2.0.0",
```

```suggestion
  "version": "2.0.0",
  "version": "1.2.0"
```
I know I initially said we might have to go up a version, but I don't think this is a breaking change (and I have plans for 2.0.0 actually being a breaking change, so I don't want to cause too much noise). Think we can stay in the 1s!

---

I messed up my suggesting by including 2 versions :/ would you mind applying this new one?